docs/content/posts/2025-02-TBD-release-1.20.1.md

```diff
@@ -0,0 +1,213 @@
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-37016'>FLINK-37016</a>] -         NPE when ClusterEntrypoing is shut down before initialization
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-37025'>FLINK-37025</a>] -         Periodic SQL watermarks can travel back in time
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-37029'>FLINK-37029</a>] -         Fix the materialized table docs of full mode cron expression
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-37084'>FLINK-37084</a>] -         NullPointerException occurs during handling timer of window in PyFlink
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-37098'>FLINK-37098</a>] -         Can not select time attribute from a view
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-37168'>FLINK-37168</a>] -         TimerRegistrationAction in unregisteredTimers is not cleaned up properly after the timers are registered
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-37183'>FLINK-37183</a>] -         Usrlib symlinks are not followed
+</li>
+</ul>
+                
+<h2>        Improvement
+</h2>
+<ul>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-36021'>FLINK-36021</a>] -         Delegating the responsibility for compression to every tier
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-36127'>FLINK-36127</a>] -         Support sorting watermark on flink web
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-36593'>FLINK-36593</a>] -         Upgrade io.airlift:aircompressor to mitigate CVE
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-36643'>FLINK-36643</a>] -         Upgrade aws-java-sdk-core to 1.12.779 to fix vulnerability
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-36689'>FLINK-36689</a>] -         [UI] Upgrade runtime web Angular framework and associated deps to latest version
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-36716'>FLINK-36716</a>] -         Address vulnerabilities in Flink UI
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-37099'>FLINK-37099</a>] -         Document possible Netty4 memory related configurations
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-37100'>FLINK-37100</a>] -         Fix test_netty_shuffle_memory_control.sh with Netty4 RPC
+</li>
+</ul>
+                                                                                                                    
+<h2>        Technical Debt
+</h2>
+<ul>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-28091'>FLINK-28091</a>] -         Thread leak in CheckpointResourcesCleanupRunnerTest#testCancellationAfterStart
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-35512'>FLINK-35512</a>] -         ArtifactFetchManagerTest unit tests fail
```

Comment:
IIRC, things related to testing shouldn't be listed here.

Reply:
+1, FLINK-35512, FLINK-28091, FLINK-37100, FLINK-36591, FLINK-36191 and FLINK-35482 can be removed.